code/obj/nuclearreactor/nuclearreactor.dm

```diff
@@ -0,0 +1,666 @@
+/////////////////////////////////////////////////////////////////
+// Defintion for the nuclear reactor engine
+/////////////////////////////////////////////////////////////////
```

Comment:
```suggestion
// ----------------------------------------------------- //
// Defintion for the nuclear reactor engine
// ----------------------------------------------------- //
```
fucks up code docs otherwise